components/cards/SpeakerCard.tsx

```diff
@@ -1,14 +1,98 @@
+import { useTheme } from '@react-navigation/native';
+import { Image } from 'expo-image';
+import { Link, useRouter } from 'expo-router';
 import React from 'react';
-import { View } from 'react-native';
+import { Dimensions, StyleSheet, TouchableOpacity, View } from 'react-native';
+import type { Session, Speaker } from '../../global';
+import { truncate } from '../../util/helpers';
+import Space from '../common/Space';
+import StyledText from '../common/StyledText';
 
-// TODO: Implement speaker card.
-/**
- * - each speaker card should display Image of speaker, name, description, and session buttons
- * - clicking session button should navigate to [session] page
- */
+const { width } = Dimensions.get('window');
 
-const SpeakerCard = () => {
-  return <View />;
+interface SpeakerCardProps extends Speaker {
+  sessions: Array<Session>;
+}
+
+const SpeakerCard = (props: SpeakerCardProps) => {
+  const { colors } = useTheme();
+  const router = useRouter();
+
+  const { name, avatar, tagline, sessions } = props;
+
+  return (
+    <View style={[styles.card, { backgroundColor: colors.card }]}>
+      <Image source={{ uri: avatar }} style={[styles.image, { borderColor: colors.tint }]} contentFit="contain" />
+
+      <Space size={16} />
+
+      <Link href={{ pathname: `/${name}`, params: { name: name } }}>
+        <StyledText font="bold" size="base" variant="primary">
+          {name}
+        </StyledText>
+      </Link>
+
+      <Space size={8} />
+
+      <View style={styles.tagline}>
+        <StyledText size="sm" variant="text" style={{ color: colors.textLight }}>
+          {truncate(50, tagline)}
+        </StyledText>
+      </View>
+
+      <Space size={16} />
+
+      {sessions.length > 0 && (
+        <TouchableOpacity
+          style={[styles.button, { borderColor: colors.tint }]}
+          onPress={() =>
+            router.push({
+              pathname: `/session/${sessions[0]?.slug}`,
+              params: { slug: sessions[0]?.slug },
+            })
+          }
+          testID="sessionButton"
+        >
+          <StyledText size="base" font="semiBold" style={[styles.buttonText, { color: colors.tint }]}>
+            Session
+          </StyledText>
+        </TouchableOpacity>
+      )}
+    </View>
+  );
```

Comment:
We should align the session buttons uniformly to the bottom of the card

<img width="365" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/42f88407-cb3c-463a-a306-2eeb84213550">